composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
-        "orchestra/testbench": "~3.0"
+        "orchestra/testbench-browser-kit": "~3.4"
```

Comment:
You are right! Some tests failed using version 3.4 of testbench, but they passed using browser-kit. Probably I just needed to update phpunit to the latest version.